region/crateria/west.json

```diff
@@ -954,45 +1146,81 @@
           "from": 1,
           "to": [
             {
-              "id": 3,
-              "requires": [
-                {"or":[
-                  "canUseMorphBombs",
-                  "canUsePowerBombs"
-                ]},
-                {"obstacle":{
-                  "id": "A",
-                  "requires": [
-                    {"or":[
-                      "ScrewAttack",
-                      "canUseMorphBombs",
-                      {"canComeInCharged": {
-                        "fromNode": 1,
-                        "framesRemaining": 0,
-                        "shinesparkFrames": 90
-                      }},
-                      {"and":[
-                        "canUsePowerBombs",
+              "id": 4,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": null,
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": ["ScrewAttack"]
+                    },
+                    {
+                      "id": "B",
+                      "requires": ["ScrewAttack"]
+                    }
+                  ]
+                },
+                {
+                  "name": "Shinespark",
+                  "notable": false,
+                  "requires": null,
+                  "obstacles": [
+                    {
+                      "id": "B",
+                      "requires": [
+                        {"canComeInCharged": {
+                          "fromNode": 1,
+                          "framesRemaining": 0,
+                          "shinesparkFrames": 90
+                        }}
+                      ],
+                      "additionalObstacles": ["A"]
+                    }
+                  ]
+                },
+                {
+                  "name": "Morph Bomb Gauntlet E-Tank Room (In)",
+                  "notable": true,
+                  "requires": null,
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": ["canUseMorphBombs"],
+                      "note": "FIXME health?"
```

Comment:
door to door is possible damageless in both directions

Reply:
It is, I think the issue at hand is leniency.
I would think in the end we'll want the damageless strat to be notable, with another one having some acid frames of leniency.